src/core/Board.ts

```diff
@@ -45,6 +47,13 @@ class Board {
 
     this.board[to.x][to.y] = piece;
     this.board[from.x][from.y] = undefined; 
+
+    /* limit move policies to one step forward */
+    if(piece.type === PieceType.Pawn) {
+      const movePolicies = [new MovePolicy(MoveType.Vertical, new MoveConstraints({max: 2})),
```

Comment:
Probably should be `{max: 1}` here :D

Reply:
Yes, probably, thanks for noting it. Will fix it.